e2e/pages/citizen/createCase/C100/otherProceedings/EmergencyProtectionDocumentUploadPage.ts

```diff
@@ -128,17 +126,19 @@ export class EmergencyProtectionDocumentUploadPage {
   private static async fillInFields({
     page: page,
   }: fillInFieldsOptions): Promise<void> {
-    const fileInput = page.locator(`${UniqueSelectors.documentUpload}`);
+    const fileInput = page.locator(
+      `${CitizenOtherProceedingsDocumentUploadSelectors.documentUpload}`,
+    );
     await fileInput.setInputFiles(config.testPdfFile);
     await page.click(
       `${Selectors.GovukButton}:text-is("${CommonStaticText.uploadFile}")`,
     );
     await page.waitForSelector(
-      `${UniqueSelectors.uploadConfirmationSelector}`,
+      `${CitizenOtherProceedingsDocumentUploadSelectors.uploadConfirmationSelector}`,
```

Comment:
I think timeout 6000 could be more sturdy because of javascript maths and playwright being so fast